locale/locales/en.json

```
@@ -7,6 +7,10 @@
     "value": "Rows per page",
     "comment": "label for the rows per page dropdown. (tew 210929)"
   },
+  "SNTable.Pagination.DisplayedRowsLabel": {
+    "value": "{0} of {1}",
+    "comment": "label for the displayed rows"
```

```suggestion
    "comment": "label for the displayed rows, e.g \"1-100 of 234\""
```

Not sure if that is how you do " insede the string, but a think an example helps here.

---

@haxxmaxx  How about   `, e.g '1-100 of 234'` ?